app/models/financial_aid/my_awards.rb

```diff
@@ -233,7 +252,12 @@ def alert_details(item_type)
     end
 
     def disbursements(item_type)
-      disbursements ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements(@uid, my_aid_year, item_type)
+      if item_type
+        disbursements ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements(@uid, my_aid_year, item_type)
```

Comment:
@marcpechaitis - You'd have to do the same thing here, because the `item_type` changes. 

If `item_type` is sometimes `nil`, then that's still something that can be stored in the hash.
```
a = {}
a[nil] = '1'
a
 => {nil=>"1"}
```

Since you have to call a different function for the value when it's `nil`... I'd do it like this:
```
def disbursements(item_type = nil)
  @disbursements ||= {}
  if item_type.present?
    @disbursements[item_type] ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements(@uid, my_aid_year, item_type)
  else
    @disbursements[item_type] ||= EdoOracle::FinancialAid::Queries.get_awards_disbursements_tuition_fee_remission(@uid, my_aid_year)
  end
end
```